pipeline/tasks/python_grpc_tasks.py

```diff
@@ -56,9 +56,12 @@ def execute(self, src_proto_path, import_proto_path, common_protos_yaml):
         with open(common_protos_yaml) as common_protos_file:
             common_protos_data = yaml.load(common_protos_file)
 
-        # Treat google.protobuf as a common proto package, even though it is
-        # not included in the common-protos we generate.
-        common_protos = ['google.protobuf']
+        # Treat google.protobuf, google.iam as a common proto package, even
+        # though they are not included in the common-protos we generate.
+        #
+        # TODO (geigerj): remove 'google.iam' when it is included in the common
+        # protos package.
+        common_protos = ['google.protobuf', 'google.iam']
```

Comment:
No -- `artman` has knowledge of the common protos *package* and which proto packages are contained within via https://github.com/googleapis/googleapis/blob/master/gapic/packaging/common_protos.yaml; `toolkit` doesn't know about this config file. There is a TODO in the `toolkit` code to fix this.